baxter_collaboration_lib/test/test_robot_arm.test

```diff
@@ -0,0 +1,6 @@
+<launch>
+    <!-- Let's load the baxter URDF from the parameter server -->
+    <include file="$(find baxter_collaboration_lib)/launch/baxter_urdf.launch" />
+
+    <test test-name="test_robot_arm" pkg="baxter_collaboration_lib" type="test_robot_arm" />
+</launch>
```

Comment:
I would change the filenames (and the executable) from `test_robot_arm` to `test_arm_ctrl`